tests/test_playwright.py

```diff
@@ -55,65 +61,39 @@ def _check_test_site(site_name: str, site_path: Path, test_func: Callable):
         test_sites_dir.rmdir()
 
 
-def test_version_switcher_highlighting(page: Page, url_base: str) -> None:
+# ------------------------- Test functions: style -------------------------
+def test_version_switcher_highlighting(
+    sphinx_build_factory: Callable, page: Page, url_base: str
+) -> None:
     """
     In sidebar and topbar - version switcher should apply highlight color to currently
     selected version.
     """
-    page.goto(url=url_base)
-    # no need to include_hidden here ↓↓↓, we just need to get the active version name
-    button = page.get_by_role("button").filter(has_text="dev")
-    active_version_name = button.get_attribute("data-active-version-name")
-    # here we do include_hidden, so sidebar & topbar menus should each have a
-    # matching entry:
-    entries = page.get_by_role("option", include_hidden=True).filter(
-        has_text=active_version_name
-    )
-    assert entries.count() == 2
-    # make sure they're highlighted
-    for entry in entries.all():
-        light_mode = "rgb(10, 125, 145)"  # pst-color-primary
-        # dark_mode = "rgb(63, 177, 197)"
-        expect(entry).to_have_css("color", light_mode)
-
-
-def test_breadcrumb_expansion(page: Page, url_base: str) -> None:
-    """Test breadcrumb text-overflow."""
-    # wide viewport width → no truncation
-    page.set_viewport_size({"width": 1440, "height": 720})
-    page.goto(urljoin(url_base, "community/topics/config.html"))
-    expect(page.get_by_label("Breadcrumb").get_by_role("list")).to_contain_text(
-        "Update Sphinx configuration during the build"
-    )
-    el = page.get_by_text("Update Sphinx configuration during the build").nth(1)
-    expect(el).to_have_css("overflow-x", "hidden")
-    expect(el).to_have_css("text-overflow", "ellipsis")
-    assert not _is_overflowing(el)
-    # narrow viewport width → truncation
-    page.set_viewport_size({"width": 150, "height": 720})
-    assert _is_overflowing(el)
-
-
-def test_breadcrumbs_everywhere(
-    sphinx_build_factory: Callable, page: Page, url_base: str
-) -> None:
-    """Test breadcrumbs truncate properly when placed in various parts of the layout."""
-    site_name = "breadcrumbs"
+    site_name = "version_switcher"
     site_path = _build_test_site(site_name, sphinx_build_factory=sphinx_build_factory)
 
-    def check_breadcrumb_truncation():
-        page.goto(
-            urljoin(url_base, f"playwright_tests/{site_name}/hansel/gretel/house.html")
+    def check_version_switcher_highlighting():
+        page.goto(urljoin(url_base, f"playwright_tests/{site_name}/index.html"))
+        # no need to include_hidden here ↓↓↓, we just need to get the active
+        # version name
+        button = page.get_by_role("button").filter(has_text="dev")
```

Comment:
I mentioned this before, but this will only be correct if the current `__version__` is a `dev` version, won't it? I.e. a release version won't have the `dev` entry as the first selected entry, so this locator will fail to find the switcher, which will manifest as an exception when we try to extract the attribute in the line below.

Reply:
Yes, but we can ensure this _always_ is `dev`. See the comment below.